docs/Change_Log.md

```diff
@@ -1,5 +1,13 @@
 # Change Log
 
+## Team Member Roles
+
+#### Aug 23, 2023
+
+You can now select roles other than admin when inviting users or configuring members of a team. There are four [role types](#DOCS_TOPICS_TEAMS/team-member-roles-team-member-role-types) that a team member can be assigned: owner, admin, developer, or read-only. The team member object now has an additional [`role` field](#DOCS_TOPICS_TEAMS/data-models-team-member-object), which is a string representing the member's current role.
```

Comment:
```suggestion
You can now select roles other than admin when inviting users or configuring members of a team. There are four [role types](#DOCS_TOPICS_TEAMS/team-member-roles-team-member-role-types) that a team member can be assigned to: owner, admin, developer, or read-only. The team member object now has an additional [`role` field](#DOCS_TOPICS_TEAMS/data-models-team-member-object), which is a string representing the member's current role.
```

And probably should mention that permissions is deprecated

Reply:
`permissions` was never documented so I'm not sure it makes sense to include here.